agents/node/vcxagent-core/demo/script-common.js

```diff
@@ -0,0 +1,27 @@
+const commandLineArgs = require('command-line-args')
```

Comment:
It seems this file is duplicated (here and in `agents/node/vcxagent-cli/script-common.js`)

Reply:
It is, also `logger.js` but I didn't find it enough to be good reason to create some sort of share `vcxagent-common`  js package. It's quite simple CLI boilerplate.